crates/vm/levm/src/gas_cost.rs

```diff
@@ -0,0 +1,688 @@
+pub const RETURNDATACOPY_STATIC: U256 = U256([3, 0, 0, 0]);
+pub const RETURNDATACOPY_DYNAMIC_BASE: U256 = U256([3, 0, 0, 0]);
+pub const ADDRESS: U256 = U256([2, 0, 0, 0]);
+pub const ORIGIN: U256 = U256([2, 0, 0, 0]);
+pub const CALLER: U256 = U256([2, 0, 0, 0]);
+pub const BLOCKHASH: U256 = U256([20, 0, 0, 0]);
+pub const COINBASE: U256 = U256([2, 0, 0, 0]);
+pub const TIMESTAMP: U256 = U256([2, 0, 0, 0]);
+pub const NUMBER: U256 = U256([2, 0, 0, 0]);
+pub const PREVRANDAO: U256 = U256([2, 0, 0, 0]);
+pub const GASLIMIT: U256 = U256([2, 0, 0, 0]);
+pub const CHAINID: U256 = U256([2, 0, 0, 0]);
+pub const SELFBALANCE: U256 = U256([5, 0, 0, 0]);
+pub const BASEFEE: U256 = U256([2, 0, 0, 0]);
+pub const BLOBHASH: U256 = U256([3, 0, 0, 0]);
+pub const BLOBBASEFEE: U256 = U256([2, 0, 0, 0]);
+pub const POP: U256 = U256([2, 0, 0, 0]);
+pub const MLOAD_STATIC: U256 = U256([3, 0, 0, 0]);
+pub const MSTORE_STATIC: U256 = U256([3, 0, 0, 0]);
+pub const MSTORE8_STATIC: U256 = U256([3, 0, 0, 0]);
+pub const JUMP: U256 = U256([8, 0, 0, 0]);
+pub const JUMPI: U256 = U256([10, 0, 0, 0]);
+pub const PC: U256 = U256([2, 0, 0, 0]);
+pub const MSIZE: U256 = U256([2, 0, 0, 0]);
+pub const GAS: U256 = U256([2, 0, 0, 0]);
+pub const JUMPDEST: U256 = U256([1, 0, 0, 0]);
+pub const TLOAD: U256 = U256([100, 0, 0, 0]);
+pub const TSTORE: U256 = U256([100, 0, 0, 0]);
+pub const MCOPY_STATIC: U256 = U256([3, 0, 0, 0]);
+pub const MCOPY_DYNAMIC_BASE: U256 = U256([3, 0, 0, 0]);
+pub const PUSH0: U256 = U256([2, 0, 0, 0]);
+pub const PUSHN: U256 = U256([3, 0, 0, 0]);
+pub const DUPN: U256 = U256([3, 0, 0, 0]);
+pub const SWAPN: U256 = U256([3, 0, 0, 0]);
+pub const LOGN_STATIC: U256 = U256([375, 0, 0, 0]);
+pub const LOGN_DYNAMIC_BASE: U256 = U256([375, 0, 0, 0]);
+pub const LOGN_DYNAMIC_BYTE_BASE: U256 = U256([8, 0, 0, 0]);
+pub const CALLVALUE: U256 = U256([2, 0, 0, 0]);
+pub const CODESIZE: U256 = U256([2, 0, 0, 0]);
+pub const CODECOPY_STATIC: U256 = U256([3, 0, 0, 0]);
+pub const CODECOPY_DYNAMIC_BASE: U256 = U256([3, 0, 0, 0]);
+pub const GASPRICE: U256 = U256([2, 0, 0, 0]);
+pub const EXTCODECOPY_DYNAMIC_BASE: U256 = U256([3, 0, 0, 0]);
+pub const SELFDESTRUCT_STATIC: U256 = U256([5000, 0, 0, 0]);
+pub const SELFDESTRUCT_DYNAMIC: U256 = U256([25000, 0, 0, 0]);
+
+// Costs in gas for call opcodes (in wei)
+pub const WARM_ADDRESS_ACCESS_COST: U256 = U256([100, 0, 0, 0]);
+pub const COLD_ADDRESS_ACCESS_COST: U256 = U256([2600, 0, 0, 0]);
+pub const NON_ZERO_VALUE_COST: U256 = U256([9000, 0, 0, 0]);
+pub const BASIC_FALLBACK_FUNCTION_STIPEND: U256 = U256([2300, 0, 0, 0]);
+pub const VALUE_TO_EMPTY_ACCOUNT_COST: U256 = U256([25000, 0, 0, 0]);
+
+// Costs in gas for create opcodes (in wei)
+pub const INIT_CODE_WORD_COST: U256 = U256([2, 0, 0, 0]);
+pub const CODE_DEPOSIT_COST: U256 = U256([200, 0, 0, 0]);
+pub const CREATE_BASE_COST: U256 = U256([32000, 0, 0, 0]);
+
+pub fn exp_gas_cost(exponent: U256) -> Result<U256, OutOfGasError> {
```

Comment:
I messed up here. It is enough with the namespacing of the module, we do not need to have the `_gas_cost` prefix in the functions. Do this for all the functions.

```suggestion
pub fn exp(exponent: U256) -> Result<U256, OutOfGasError> {
```

Reply:
Done: [Change in imports + fmt](https://github.com/lambdaclass/lambda_ethereum_rust/pull/1138/commits/33b1add9cff76103495657d5ae565e0d1fad07cf)